.github/workflows/e2ePerf.yml

```diff
@@ -71,8 +71,10 @@ jobs:
             echo "The sfdx-cli installation could not be verified"
             exit 1
           fi
-      - name: Setup YourKit
-        run: echo "YOURKIT_PROFILER_AGENT=${{ env.RUNNER_OS }}-${{ env.RUNNER_ARCH}}=sampling,async_sampling_cpu,app_name=apex_lsp,alloc_object_counting" >> $GITHUB_ENV
+      - name: Setup YourKit Agent
+        run: echo "YOURKIT_PROFILER_AGENT=${{ env.RUNNER_OS }}-${{ env.RUNNER_ARCH}}=sampling,async_sampling_cpu,app_name=apex_lsp,alloc_object_counting,onexit=memory,onexit=memory,onexit=snapshot,onexit=hprof,dir=./snapshots" >> $GITHUB_ENV
```

Comment:
`onexit=memory` twice? Not sure if this was deliberate?

Reply:
@supert3d Good catch. I'll log a bug to have the duplicate removed.